torch/testing/_internal/common_fsdp.py

```diff
@@ -1521,9 +1553,11 @@ def forward(self, x):
 class SkipModel(nn.Module):
     def __init__(self, double_nest):
         super().__init__()
-        self.linear = nn.Linear(10, 10, bias=False).cuda()
-        self.linear_skip = SkipModule().cuda()
-        self.nested_linear = wrap(NestedLinear(fsdp_wrap=double_nest))
+        self.linear = nn.Linear(10, 10, bias=False).to(DEVICE_TYPE)
+        self.linear_skip = SkipModule().to(DEVICE_TYPE)
+        self.nested_linear = wrap(
+            NestedLinear(fsdp_wrap=double_nest), device_id=DEVICE_TYPE
```

Comment:
is this a behavior change? were we relying on the 'set_device' for cuda to cover this before and now we make it explicit?

Reply:
@ankurneog this